R/tbl.R

```diff
@@ -326,6 +326,29 @@ collect.tbl_kusto <- function(tbl, ...)
     as_tibble(res)
 }
 
+random_table_name <- function() {
```

Comment:
I think a better name would be `generate_table_name`; functions should be verbs.

Also, "AzureKusto_" as a prefix is doubly redundant at the table level, we already know it's in Azure and it's a Kusto table. The name makes sense for the R package because it identifies it as specifically Azure and Kusto among the 15000 others. Perhaps "Rtbl_" would be better, that identifies the table as coming from R.